encord/orm/dataset.py

```diff
@@ -120,14 +215,206 @@ def created_at(self, value: datetime) -> None:
         """Datetime will trim milliseconds for backwards compatibility."""
         self["created_at"] = value.strftime(DATETIME_STRING_FORMAT)
 
+    @property
+    def frames_per_second(self) -> Optional[int]:
+        """
+        Returns:
+        If the data type is `DataType.VIDEO` this returns the actual number of frames per second for the video.
+        Otherwise, it returns None as a `frames_per_second` field is not applicable.
+        """
+        if self.data_type != DataType.VIDEO:
+            return None
+        return self["frames_per_second"]
+
+    @property
+    def duration(self) -> Optional[int]:
+        """
+        Returns:
+        If the data type is `DataType.VIDEO` this returns the actual video duration.
+        Otherwise, it returns None as a `duration` field is not applicable.
+        """
+        if self.data_type != DataType.VIDEO:
+            return None
+        return self["duration"]
+
     @property
     def client_metadata(self) -> Optional[dict]:
         """
-        Custom client metadata. This is null if it is disabled via the
-        :class:`encord.orm.dataset.DatasetAccessSettings`
+        DEPRECATED - please use `get_client_metadata` instead.
```

Comment:
This is marked as deprecated, but the implementation changed which we cannot do for current clients.